### PR TITLE
feat: add supabase-edge-functions example

### PR DESCRIPTION
**Pull request by thorwebdev**
Thanks for building grammY! Was super easy to get it running with Deno on Supabase 🥳

### CODE REVIEW

File: supabase-edge-functions/supabase/functions/telegram-bot/index.ts [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    return await handleUpdate(req);
+  } catch (err) {
+    console.error(err);
+  }
+})
+
+// To invoke:
+// curl -i --location --request POST 'http://localhost:54321/functions/v1/' \
+//   --header 'Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJzdXBhYmFzZS1kZW1vIiwicm9sZSI6ImFub24ifQ.625_WdcF3KHqz5amU0x2X5WWHP-OEs_4qj0ssLNHzTs' \
```
Comment (KnorpelSenf) [inline supabase-edge-functions/supabase/functions/telegram-bot/index.ts:33]: Is this a leaked token?

**Review by KnorpelSenf — CHANGES_REQUESTED** [review]

File: supabase-edge-functions/supabase/functions/telegram-bot/index.ts (hunk 1 above)
Comment (thorwebdev) [reply supabase-edge-functions/supabase/functions/telegram-bot/index.ts:33]: No, it's a test token which can be used for local testing. But good point, this example doesn't actually need an auth header, so I'll remove, thanks.

Comment (thorwebdev) [reply supabase-edge-functions/supabase/functions/telegram-bot/index.ts]: Removed

**Review by KnorpelSenf — APPROVED** [review]
LGTM. Thanks!

**Pull request merged**